gematria/datasets/convert_bhive_to_llvm_exegesis_input.cc

```diff
@@ -64,6 +65,14 @@ int main(int argc, char* argv[]) {
       gematria::ConvertHexToString(kInitialRegVal);
   std::string initial_mem_val_str =
       gematria::ConvertHexToString(kInitialMemVal);
+  // Prefix the string with zeroes as llvm-exegesis assumes the bit width
```

Comment:
this is unrelated?

Reply:
Slightly unrelated, but necessary to make this patch work properly. We're assuming the memory value has a width of 64 bits, but only setting the first x bits. Exegesis determines the width of the value based on how many characters are in the string, so we need to explicitly print out the leading zeros.